### PR TITLE
OSDOCS-4872: change variant rhcos to openshift

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OSDOCS-4872

Link to docs preview: http://file.rdu.redhat.com/jdohmann/OSDOCS-4872/installing/installing_sno/install-sno-installing-sno.html#create-custom-live-rhcos-iso_install-sno-installing-sno-with-the-assisted-installer

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by jldohmann** [comment]
@mike-nguyen could you PTAL for QE ack? this updates the SNO docs to reflect the changes in Butane support. it doesn't look like these docs existed pre-4.12, so i've marked them as 4.12+

**Comment by mike-nguyen** [comment]
I'm not too familiar with single node mode so I can't comment on the dropping of root ssh login.  If SNO is more like OCP now with only allowing core user login then the changes LGTM. Do you know if SNO has QE that can look over these changes?

File: modules/creating-custom-live-rhcos-iso.adoc [hunk 1]
```diff
@@ -22,67 +22,36 @@ You can create a live {op-system} ISO with SSHd enabled and with predefined cred
...
-storage:
-  files:
-    - path: /etc/ssh/sshd_config
-      contents:
-        local: ./live-sshd_config
-      mode: 0600
-      overwrite: true
 ----
-<1> `yescrypt` passwords are not supported in {op-system}. Use `bcrypt` instead when you create the `password_hash`.
+<1> The `core` user has sudo privileges.
```
Comment (maxwelldb) [inline modules/creating-custom-live-rhcos-iso.adoc:39]: Has or must have?

File: modules/creating-custom-live-rhcos-iso.adoc [hunk 2]
```diff
@@ -22,67 +22,36 @@ You can create a live {op-system} ISO with SSHd enabled and with predefined cred
...
-
-. Pass the YAML file and the folder to the `butane` utility:
-+
-[source,terminal]
-----
-$ butane -p embedded.yaml -d files-dir/ > embedded.ign
-----
-
-. Once the Ignition file is created, you can include the configuration in a new live {op-system} ISO, which is named `rhcos-sshd-4.12.0-x86_64-live.x86_64.iso`, with the `coreos-installer` utility:
+. Once the Ignition file is created, you can include the configuration in a new live {op-system} ISO, which is named `rhcos-sshd-{product-version}.0-x86_64-live.x86_64.iso`, with the `coreos-installer` utility:
```
Comment (maxwelldb) [inline modules/creating-custom-live-rhcos-iso.adoc:41]: I know it's just the existing language, but:

```suggestion
. After the Ignition file is created, you can include the configuration in a new live {op-system} ISO, which is named `rhcos-sshd-{product-version}.0-x86_64-live.x86_64.iso`, with the `coreos-installer` utility:
```

⚠️ ISG

File: modules/creating-custom-live-rhcos-iso.adoc [hunk 3]
```diff
@@ -22,67 +22,36 @@ You can create a live {op-system} ISO with SSHd enabled and with predefined cred
...
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
-$ coreos-installer iso ignition embed -i embedded.ign rhcos-4.12.0-x86_64-live.x86_64.iso -o rhcos-sshd-4.12.0-x86_64-live.x86_64.iso
+$ coreos-installer iso ignition embed -i embedded.ign rhcos-{product-version}.0-x86_64-live.x86_64.iso -o rhcos-sshd-{product-version}.0-x86_64-live.x86_64.iso
 ----
 
 .Verification
 
 . Check that the custom live ISO can be used to boot the server by running the following command:
```
Comment (maxwelldb) [inline modules/creating-custom-live-rhcos-iso.adoc:50]:
```suggestion
* Check that the custom live ISO can be used to boot the server by running the following command:
```

One-step procedures should be a UL.

**Review by maxwelldb — COMMENTED** [review]
Looking pretty good. One question for you, as well as two suggested changes for while you're working on the file.

File: modules/creating-custom-live-rhcos-iso.adoc (hunk 1 above)
Comment (jldohmann) [reply modules/creating-custom-live-rhcos-iso.adoc:39]: Has 🙂 coreos `core` user comes with special admin privs by default, but still needs to be configured for actual use with ssh keys etc. just like you'd configure other users, so it's worth calling out here to note differences

**Comment by jldohmann** [comment]
4621c33..aeb7429910 implement peer review feedback
aeb7429910..290499cb97 rebase

**Comment by jldohmann** [comment]
@yunjiang29 could you PTAL for QE ack, or do you know who could take a look for QE ack? this PR updates the SNO docs to reflect the changes in Butane support. thank you!

File: modules/creating-custom-live-rhcos-iso.adoc [hunk 4]
```diff
@@ -18,71 +18,47 @@ You can create a live {op-system} ISO with SSHd enabled and with predefined cred
...
-The modified `sshd_config` file overrides the default `sshd_config` file.
-====
-
-. Pass the YAML file and the folder to the `butane` utility:
+. Run the `butane` utility to create the Ignition file using the following command:
 +
 [source,terminal]
 ----
-$ butane -p embedded.yaml -d files-dir/ > embedded.ign
+$ butane -pr embedded-rhcos.yaml -o embedded-rhcos.ign
```
Comment (sgoveas) [inline modules/creating-custom-live-rhcos-iso.adoc:45]: Sorry, please remove `-rhcos` from the file names. I added this string while testing. The name of files `embedded.yaml` and `embedded.ign` should match the previous and later mentions.

Same for `-p (pretty format)`. The command should be
`$ butane -r embedded.yaml -o embedded.ign`

Comment (jldohmann) [reply modules/creating-custom-live-rhcos-iso.adoc]: updated, thanks for the correction!

**Review by sgoveas — APPROVED** [review]
/lgtm

**Comment by jldohmann** [comment]
I am leaving the hardcoded 4.12 version in the `link` macro but will apply cherrypicks with a CIT to return back and fix for each enterprise branch. The issue with using `{product-version}` in its place is detailed [here](https://github.com/redhataccess/ascii_binder/pull/180)

**Pull request merged**